app/assets/locales/en.json

```diff
@@ -23,6 +23,7 @@
   "are_you_sure": "Are you sure?",
   "return_home": "Return Home",
   "created_at": "Created at",
+  "search_not_found": "No {{ name }} found",
```

Comment:
It's normally not a good idea to pass in the text here because this becomes unlocalizable. In english it might be "no rooms found" but in other languages it might be something like "not found room" (just and example)

Reply:
We're better off precreating all of the different options